src/app/routes/utils/sharedDataTransformers/augmentWithTimestamp/index.test.js

```diff
@@ -116,6 +116,7 @@ describe('Timestamp rules', () => {
       metadata: {
```

Comment:
We should a new test case for the STY logic we've added to `augmentWithTimestamp`